src/shell/components/FieldTypeDateTime/util.ts

```diff
@@ -98,6 +98,1376 @@ const TIME_STRINGS = [
   "11:30 pm",
   "11:45 pm",
 ] as const;
+export const TIMEZONES = [
```

Comment:
@shrunyan moment-timezone only provides the timezone id not the timezone name/label

Reply:
They offer a data pack of timezone info. Any reason we are not using this? https://momentjs.com/timezone/docs/#/use-it/

Reply:
@shrunyan ohhh I didn't notice that. I'll give it a look later today and see if we can use those packages.

Reply:
@shrunyan tried checking the data pack but I can't find a way to generate the standard name for each timezone using it.  Since the standard names aren't declared anywhere in moment-tz. These standard names are being used as the user-friendly labels on the dropdown instead of just showing them the timezone IDs directly.

Example: converting Asia/Manila -> (GMT+8:00) Philippine Standard Time or Asia/Dili -> (GMT+09:00) East Timor Time